### PR TITLE
Added a parameter 'use_hpo'

### PR DESCRIPTION
**Pull request by priyanka9991**
use_hpo = True : Model uses HPO optimized learning rate and batch size.
use_hpo = False: Model uses default values of learning rate and batch size.
hyperparameters.json file also contains a provision to add default values of HPs.

### CODE REVIEW

**Review by wilke — DISMISSED** [review]
Those changes are unnecessary. The model should always use the provided config file, and the workflow should set the parameters explicitly instead of relying on unknown defaults. From an experiment point of view, I want to control what parameters the model is using; hence, I define them as input for the workflow outside of the code.

**Comment by priyanka9991** [comment]
@wilke  I have removed the 'use_hpo' parameter. The users can provide the name of their hyperparameters json file in the csa_params.ini file. In this directory, I have provided two such files: hyperparameters_hpo.json and hyperparameters_default.json as templates. The users can input any file of their choice by changing it in csa_params.ini.

File: workflows/parsl_csa/hyperparameters_default.json
Comment (wilke) [inline workflows/parsl_csa/hyperparameters_default.json:1]: I assume this file is not complete yet?

**Review by wilke — APPROVED** [review]
LGTM

File: workflows/parsl_csa/hyperparameters_default.json
Comment (priyanka9991) [reply workflows/parsl_csa/hyperparameters_default.json:1]: Yes. Waiting for default parameters from all the models.
